stdlib/std.jsonnet

```diff
@@ -1004,13 +1004,13 @@ limitations under the License.
           std.join('\n' + cindent, lines);
     aux(value, [], ''),
 
-  manifestYamlStream(value, indent_array_in_object=false)::
+  manifestYamlStream(value, indent_array_in_object=false, disable_c_document_end=false)::
```

Comment:
It is not really important, but perhaps instead of `disable_c_document_end=false`, `c_document_end=true` or `with_c_document_end=true` would be more natural (right now it's a bit like double negation).

Reply:
I think the reason I did it that way is that I'm used to proto3 messages where the default is always false.  Any idea if the python style guide says anything about preferring falsey defaults?

Reply:
I don't think Python style guide says anything about that. Python standard library uses both True and False as default values. 

In case of proto3 I guess it mostly has to do with interoperability between versions and with protobuf not sending these default values over the wire (which is a good reason to have one fixed default value per type or else the sender's default may be changed to the receiver's default).  This is a concern for over-the-wire protocols, but not really for a library function.

Reply:
Done.